structinf.go

```diff
@@ -15,6 +15,12 @@ type structCBORSpec struct {
 	strKeyForField map[string]string
 }
 
+// TaggedElement is used to wrap elements which may be tagged for writing.
+type TaggedElement struct {
+	tag   CBORTag
```

Comment:
If `TaggedElement` is exported then it's fields should be public so other packages can set them.